src/ages/ages.types.ts

```diff
@@ -0,0 +1,30 @@
+import { BigNumber, providers } from "ethers";
+import { Optional } from "../helpers/types";
+import { MarketEmission } from "../utils";
+
+export interface EpochConfig {
+  id: string;
+  snapshotBlock?: number;
+  initialTimestamp: BigNumber;
+  finalTimestamp: BigNumber;
+  initialBlock?: number;
+  finalBlock?: number;
+  totalEmission: BigNumber;
+  epochName: string;
+}
+export interface Epoch2Config extends EpochConfig {
+  compoundRepartition: BigNumber; // the percentage of tokens emitted on Compound in Base units
+  aaveRepartition: BigNumber; // the percentage of tokens emitted on Aave in Base units
+}
```

Comment:
Perhaps would it be cleaner to have this in EpochConfig instead of defining a new type, under a `protocolDistribution` key:

Reply:
```suggestion
export interface EpochConfig {
  ...
  protocolDistribution: {
    [protocol: string]: BigNumber; // the percentage of tokens distributed to users of Morpho-protocol in bps
  };
}
```

Also, I don't think "base unit" is a standard naming for basis points (googling it leads to nothing)

Reply:
Mmmh the problem with that is that we can not predict if it will be used for each next Age. I'll do it because it seems cleaner but It is sth to keep in mind